mysite/urls.py

```diff
@@ -18,5 +17,5 @@
 
 urlpatterns = [
     path('kvent/', include('Kvent.urls')),
-    path('admin/', admin.site.urls),
-]
+    path('admin/', admin.site.urls)
+]
```

Comment:
You should add all of the auth paths here:
`path('accounts/', include('django.contrib.auth.urls')),`